src/Templates/parts/table-content.twig

```diff
@@ -18,7 +18,7 @@
                 <a href="{{column.url}}" title="{{column.title}}"><span class="glyphicon glyphicon-{{column.value}}">&nbsp;</span></a>
         {% endif %}
       {% else %}
-                {{column | raw}}
+                {{column}}
```

Comment:
I remember that this will break something, but I can't remember what. Helpful, I know.